server/src/main/java/org/candlepin/controller/ContentAccessManager.java

```diff
@@ -247,44 +247,35 @@ public ContentAccessManager(
      * @throws IOException thrown if there's a problem reading the cert.
      * @throws GeneralSecurityException thrown security problem
      */
-    public ContentAccessCertificate getCertificate(Consumer consumer)
-        throws GeneralSecurityException, IOException {
```

Comment:
If these exceptions are not on the method contract let's also remove them from the javadoc above

Reply:
done